docs/user/04-metrics.md

```diff
@@ -1,3 +1,395 @@
+If a MetricPipeline configures a feature in the `input.application` section, an additional DaemonSet gets deployed acting as an agent. The agent is as well based on an [OTel Collector](https://opentelemetry.io/docs/collector/) and is encompassing the collection and conversion of prometheus-based metrics. Hereby, the workload simply puts an typical `prometheus.io/scrape` annotation onn the pod or service specification and the agent will collect scraping it. The agent will push all data in OTLP to the central gateway.
+### Telemetry Manager
+The MetricPipeline resource is managed by Telemetry Manager, a typical Kubernetes [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) responsible for managing the custom parts of the OTel Collector configuration.
+
+![Manager resources](./assets/metrics-resources.drawio.svg)
+
+Telemetry Manager watches all MetricPipeline resources and related Secrets. Whenever the configuration changes, it validates the configuration and generates a new configuration for the gateway and agent, where for each a ConfigMap for the configuration is generated. Referenced Secrets are copied into one Secret that is mounted to the gateway as well.
+Furthermore, the manager takes care of the full lifecycle of the Gateway Deployment and the Agent DaemonSet itself. Only if there is a MetricPipeline defined, they are deployed. At anytime, you can opt out of using the feature by not specifying a MetricPipeline.
+
+## Setting up a MetricPipeline
+
+In the following steps, you can see how to set up a typical MetricPipeline. Learn more about the available [parameters and attributes](resources/05-metricpipeline.md).
+
+### Step 1. Create a MetricPipeline with an output
+1. To ship metrics to a new OTLP output, create a resource file of the kind `MetricPipeline`:
+
+   ```yaml
+   apiVersion: telemetry.kyma-project.io/v1alpha1
+   kind: MetricPipeline
+   metadata:
+     name: backend
+   spec:
+     output:
+       otlp:
+         endpoint:
+           value: https://backend.example.com:4317
+   ```
+
+   This configures the underlying OTel Collector of the gateway with a pipeline for metrics. The receiver of the pipeline will be of the OTLP type and be accessible using the `telemetry-otlp-metrics` service. As an exporter, an `otlp` or an `otlphttp` exporter is used, dependent on the configured protocol.
+
+2. To create the instance, apply the resource file in your cluster:
+    ```bash
+    kubectl apply -f path/to/my-metric-pipeline.yaml
+    ```
+
+3. Check that the status of the MetricPipeline in your cluster is `Ready`:
+    ```bash
+    kubectl get metricpipeline
+    NAME              STATUS    AGE
+    http-backend      Ready     44s
+    ```
+
+### Step 2. Switch the protocol to HTTP
+
+To use the HTTP protocol instead of the default GRPC, use the `protocol` attribute and ensure that the proper port is configured as part of the endpoint. Typically, port `4317` is used for GRPC and port `4318` for HTTP.
```

Comment:
```suggestion
To use the HTTP protocol instead of the default GRPC, use the `protocol` attribute and ensure that the correct port is configured as part of the endpoint. Typically, port `4317` is used for GRPC and port `4318` for HTTP.
```